test/test_scanner.py

```diff
@@ -462,6 +462,47 @@ def test_openssh_7_9(self):
                 "CVE-2018-15473",
             ],
         )
+    @unittest.skipUnless(os.getenv("LONG_TESTS") == "1", "Skipping long tests")
+    def test_openssh_deb_7_9(self):
+        """Test detection of openssh 7.9 debian packages"""
+        print("testing debian package: openssh-client 7.9")
```

Comment:
None of the other tests have print statements. These are informative, though, so I'd recommend you switch them to comments.